client/jetpack-connect/style.scss

```
@@ -369,6 +369,14 @@ $colophon-height: 50px;
 				color: var(--jp-gray-50, #646970);
 			}
 		}
+
+		.jetpack-connect__action--onboarding-disclaimer {
+			p {
+				padding-top: 20px;
+				color: var(--jp-gray-70, #757575);
+				text-align: left;
```

It's better to use RTL aware alternative

```suggestion
				text-align: start;
```